### PR TITLE
changed ajith and soundar file

### PR DESCRIPTION
**Pull request by soucoffee1**
Hi Ajith, 
I have added the texts on soundar.txt and ajith.txt files.

### CODE REVIEW

File: soundar.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Hello Soundar
\ No newline at end of file
+Hello Soundar
+Hello Ajith
```
Comment (thinkAjith) [inline soundar.txt]: hey this can be better

File: ajith.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-Hello Ajith
\ No newline at end of file
+Hello Ajith
+Hello Nimesh
```
Comment (thinkAjith) [inline ajith.txt]: change name to amith

**Review by thinkAjith — CHANGES_REQUESTED** [review]
Please go through comments and make the changes

File: soundar.txt (hunk 1 above)
Comment (soucoffee1) [reply soundar.txt]: changes has been done.

File: ajith.txt (hunk 2 above)
Comment (soucoffee1) [reply ajith.txt]: changed the name.

**Review by thinkAjith — APPROVED** [review]
Approved. Nice coding.

**Pull request merged**